libs/positions/src/lib/liquidation-price.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEstimatePositionQuery } from './__generated__/Positions';
+import { formatRange } from '@vegaprotocol/utils';
+
+export const LiquidationPrice = ({
+  marketId,
+  openVolume,
+  collateralAvailable,
+  decimals,
+  quantum,
+}: {
+  marketId: string;
+  openVolume: string;
+  collateralAvailable: string;
+  decimals: number;
+  quantum: string;
+}) => {
+  const { data: currentData, previousData } = useEstimatePositionQuery({
+    variables: {
+      marketId,
+      openVolume,
+      collateralAvailable,
+    },
+    fetchPolicy: 'no-cache',
+    skip: !openVolume || openVolume === '0',
+  });
+  const data = currentData || previousData;
+  let value = '-';
+
+  if (data) {
+    const bestCase =
+      data.estimatePosition?.liquidation?.bestCase.open_volume_only.replace(
+        /\..*/,
+        ''
+      );
+    const worstCase =
+      data.estimatePosition?.liquidation?.worstCase.open_volume_only.replace(
+        /\..*/,
+        ''
+      );
+    value =
+      bestCase && worstCase && BigInt(bestCase) < BigInt(worstCase)
+        ? formatRange(bestCase, worstCase, decimals, quantum, value)
```

Comment:
I need to double-check here with you we're actually using market decimals and not settlement asset decimals for `notional` and `liquidation price`, hence not using quantum


this is because on futures, the price is dependent on the quote, not on the settlement asset

Reply:
I'm updating that in deal ticket in this PR too https://github.com/vegaprotocol/frontend-monorepo/pull/4114

Reply:
I switched formatting for liquidation price from quantum to marketDecimalPlaces